test/e2e/e2e_setup_test.go

```diff
@@ -185,23 +135,126 @@ func Setup(specName string, f func(testSpecificSettings func() testSettings), op
 			Variables:            testSpecificVariables,
 		})
 
-		if testMode == SupervisorTestMode {
-			runtimeExtensionProviders = append(runtimeExtensionProviders, "vm-operator", "net-operator")
-		}
+		postNamespaceCreatedFunc = func(managementClusterProxy framework.ClusterProxy, workloadClusterNamespace string) {
+			var ipVariables map[string]string
 
-		if testTarget == VCSimTestTarget {
-			runtimeExtensionProviders = append(runtimeExtensionProviders, "vcsim")
+			if testTarget == VCSimTestTarget && options.useKindForManagementCluster {
+				Byf("Creating a vcsim server")
+				Eventually(func() error {
```

Comment:
q: why are we creating another VC sim server? is it the one running on the kind cluster?

might be we need to check that it is reconciled as something pops up in status

Reply:
This is on the second kind cluster.